src/app/components/alertifier.js

```diff
@@ -11,7 +11,7 @@ export const SALUTATION = 'Hey!';
  *
  * Usage: const myAlertifier = Alertifier(document.querySelector('#myAlertifier'));
  */
-export default Alertifier = (container) => {
+export default (container) => {
```

Comment:
Good point. But it was not working for me like this. Maybe I need to export it after declaration. Will check it.